_minutes/2022-03-03-wecg.md

```diff
@@ -0,0 +1,127 @@
+# WECG Meetings 2022, Public Notes, Mar 3
+
+ * Chair: Simeon Vincent
+ * Scribes: Rob Wu
+
+Time: 8 AM PST = https://everytimezone.com/?t=62200500,3c0
+Call-in details: [WebExtensions CG, 3rd March 2022](https://www.w3.org/events/meetings/d7bbce8f-549f-46ea-b440-ea6902f8707c/20220303T080000)
+Zoom issues? Ping @zombie (Tomislav Jovanovic) in [chat](https://github.com/w3c/webextensions/blob/main/CONTRIBUTING.md#joining-chat)
+
+
+## Agenda: [discussion in #165](https://github.com/w3c/webextensions/issues/165), [other issues](https://github.com/w3c/webextensions/issues)
+
+The meeting will start at 3 minutes after the hour.
+
+ * **Carry-over from previous meetings**
+   * None
+ * **Other new issues**
+ * **Open discussion queue (add yourself at the bottom)**
+   * [tophf] Produce evidence/statistics on why persistent background scripts are bad for performance
+   * [Issue 154](https://github.com/w3c/webextensions/issues/154): Proposal: browser.secureStorage API
+     [PR 167](https://github.com/w3c/webextensions/pull/167): Add browser.secureStorage proposal
+   * [simeon] [Chrome's Off-screen Documents Proposal](https://docs.google.com/document/d/1b-I-vXq2h7OFFmus78jZXIWcKilKJLKLeGplnY9wt7k/edit)
+     * [PR 170](https://github.com/w3c/webextensions/issues/170): Offscreen Documents Proposal
+   * [carlos] WebRequest API
+     * https://github.com/w3c/webextensions/issues/157
+     * https://github.com/w3c/webextensions/issues/164
+     * https://github.com/w3c/webextensions/issues/168
+     * https://github.com/w3c/webextensions/issues/169
+ * **Check-in on ongoing issues**
+
+
+## Attendees (sign yourself in)
+
+ 1. Jack Works (Sujitech)
+ 2. Simeon Vincent (Google)
+ 3. Timothy Hatcher (Apple)
+ 4. Rob Wu (Mozilla)
+ 5. Bradley Cushing (Dashlane)
+ 6. Oliver Dunk (1Password)
+ 7. Krzysztof Modras (Ghostery)
+ 8. Bastien Granger (Dashlane)
+ 9. Giorgio Maone (NoScript)
+ 10. Craig Lurey (Keeper)
+ 11. Carlos Jeurissen (Jeurissen Apps)
+ 12. Tomislav Jovanovic (Mozilla)
+ 13. Frederic Rivain (Dashlane)
+ 14. Todd Schiller (PixieBrix)
+ 15. Philipp Claßen (Ghostery)
+ 16. Dave Herold (Keeper)
+ 17. Zane Bond (Keeper)
+ 18. Alexei (Privacy Badger)
+ 19. Steven McLintock (1Password)
+ 20. Tyler Carson (Keeper)
+ 21. Nir Nahum (WalkMe)
+ 22. Rainer Enders (Keeper Security)
+ 23. Sergey Aldoukhov (Keeper Security)
+ 24. Mukul Purohit (Microsoft)
+ 25. Igor Oleinikov (Grammarly)
+
+
+## Meeting notes
+
+Discussion item: Evidence for performance claims about persistent background scripts
```

Comment:
As an extension maintainer, this doesn't really strike confidence that everything is going to be resolved by next year. As mentioned by `craig`, "It's unreasonable to impose deadlines and force rewrites on companies". A fake deadline to scare everyone into moving may be even less respectful. From my POV it would be better to compel developers with promises of better x, y, z, that could give products a competitive edge in their category.  

The fact that extensions can no longer be updated, without exception, and the fact that chrome is now aggressively stating in the errors section of the developer console that MV2 extensions will no longer work come next year means that one must hedge bets. 

There's a limited event page proposal that is implemented by Mozilla/Apple already that gave me some hope there might be a reasonable path forward one could follow without overdue attention, but Google (who seems to chair this committee) is already proposing something different @ https://docs.google.com/document/d/1b-I-vXq2h7OFFmus78jZXIWcKilKJLKLeGplnY9wt7k/edit

This leaves me with the impression that I'll have to hawk this evolution and prepare for a service workers only scenario. 

As part of the review process for extensions it's already standard to have to justify use of permissions. Currently, it takes very little extra work (in the scheme of things) to support multiple platforms with an extension. The future is now uncertain.